plugins/Login/Security/BruteForceDetection.php

```diff
@@ -36,6 +36,11 @@ public function __construct(SystemSettings $systemSettings)
 
     public function isEnabled()
     {
+        $module = Common::getRequestVar('module', false);
```

Comment:
@diosmosis not sure about that way... eg you could maybe add a `&module=CoreUpdater` while maybe somehow still pinging the API or UI or archiving? like requesting `http://.../misc/user/archive.php?module=CoreUpdater`. I wonder if there's a different way to find out if we are currently "updating" or whether the `coreupdater` is actually being rendered

Reply:
good point

Reply:
I guess we could check `Updater::isNewVersionAvailable()` but I don't think we'd want to do that on every request...

Or could check that the version_core option >= 3.8.0. If version_core is autoloaded, then that would be ok I think...

Reply:
@tsteur went w/ disabling it if version_core is < 3.8.0. should be the fastest approach. after 3.8 the tables should be there, so wouldn't this problem to arise.